apps/src/templates/projects/generateFakeProjects.js

```diff
@@ -41,17 +41,17 @@ export const stubFakePersonalProjectData = [
   },
 ];
 
-export const stubFakeFeaturedProjectData = [
+export const stubFakeActiveFeaturedProjectData = [
   {
-    projectName: 'Featured Project 1',
+    projectName: 'Active Featured Project 1',
     type: 'gamelab',
     channel: 'abcd5',
     publishedAt: '2015-02-04T23:59:59.999-08:00',
     featuredAt: '2016-12-31T23:59:59.999-08:00',
     unfeaturedAt: 'null',
   },
   {
-    projectName: 'Featured Project 2',
+    projectName: 'Activie Featured Project 2',
```

Comment:
tiny nit: typo "Activie" -> "Active"

Are "active" and "featured" synonymous? meaning they'll show in the public gallery?

Reply:
> Are "active" and "featured" synonymous? meaning they'll show in the public gallery?

Yes -that's correct.